packages/create/templates/hello-v2/src/operations.ts

```diff
@@ -0,0 +1,64 @@
+// Welcome to DBOS!
+
+// This is a sample "Hello" app built with DBOS.
+// It greets visitors and keeps track of how many times each visitor has been greeted.
+
+// First, let's import DBOS
+import { DBOS } from "@dbos-inc/dbos-sdk";
+
+// Then, let's declare a type representing the "dbos_hello" database table
```

Comment:
Can this just replace hello completely? We should either remove or upgrade the old templates...

Reply:
We were thinking at some point in time, but that we're not really ready to handle the external name use yet.  Once there is doc, etc., then getting rid of old hello makes sense.
This could be a week or two, but the PR is already pretty big and probably shouldn't hang out that long.